syft/pkg/cataloger/debian/package.go

```diff
@@ -17,9 +18,10 @@ import (
 )
 
 const (
-	md5sumsExt   = ".md5sums"
-	conffilesExt = ".conffiles"
-	docsPath     = "/usr/share/doc"
+	md5sumsExt      = ".md5sums"
+	conffilesExt    = ".conffiles"
+	controlFilesExt = ".control"
```

Comment:
Support parsing the control files which are similar to Debian files. This was added for OpenWrt package files